addons/im_livechat/static/src/messaging/component/thread_icon/thread_icon.xml

```diff
@@ -3,8 +3,16 @@
     <t t-inherit="mail.messaging.component.ThreadIcon" t-inherit-mode="extension">
         <xpath expr="//*[@name='rootCondition']" position="inside">
             <t t-elif="thread.channel_type === 'livechat'">
-                <!-- TODO FIXME add typing icon -->
-                <div class="fa fa-comments" title="Livechat"/>
+                <t t-if="thread.typingMembers.includes(thread.directPartner)">
```

Comment:
same remark as the DM: would be better if the condition was generic, based on the presence of at least one typing member not being current partner